### ISSUE TITLE
Add hashing for credential storage

### ISSUE THREAD

**Issue by 0xkasun** [scorelab/OpenDF#42]
Currently the username and passwords are stored as plain text in the database. (except in the first instance when the administrator is created). The passwords can hashed  and salted. (a separate table has to be created to store the salts)

**Comment by agentmilindu** [comment]
Yes Kasun, we have to fix this. We can use bcrypt password hashing as @lucasjones did in #24
